core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3272,6 +3272,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_NETWORK_BLOCK_MOVER_THREADS_MAX =
```

Comment:
I think so.It might be better to add a switch to select whether this feature is enabled or not ? And we can adjust the number of threads in a controllable range.